src/core/include/mp-units/quantity.h

```diff
@@ -365,7 +386,9 @@ class quantity {
 
 // CTAD
 template<QuantityLike Q>
-explicit quantity(Q) -> quantity<quantity_like_traits<Q>::reference, typename quantity_like_traits<Q>::rep>;
+explicit(
+  is_specialization_of<decltype(quantity_like_traits<Q>::to_numerical_value(std::declval<Q>())), convert_explicitly>)
+  quantity(Q) -> quantity<quantity_like_traits<Q>::reference, typename quantity_like_traits<Q>::rep>;
```

Comment:
It seems like CTAD isn't tested for `convert_implicitly`.
I suspect these wouldn't work:
`quantity{1s}`,
`quantity_point{std::chrono::sys_seconds{1s}}`.

Reply:
Plenty of tests are still missing ;-)

Reply:
But I am not sure why this would fail?

Reply:
You're right, it seems like it would work.
I was looking at the explicit specifier and thinking "`convert_implicitly` isn't covered".
But of course it wouldn't.